### DISCUSSION TITLE
自定义圆形combo 设置半径大小不生效

### DISCUSSION THREAD

**Discussion by huangyq199205** [antvis/G6#3387] [Show and tell]
G6.registerCombo(
  'cCombo',{
    draw(cfg, group) {
      const { label, asset, system} = cfg;
      const circle = group.addShape('circle', {
        attrs:{
          r: 120,
          fill: '#999',
        },
        name:'areaCombo-shape'
      })      
      return circle
    }
  }
, 'circle');
期望： combo如果设定大小了，可以指定combo 大小 而不是根据子节点自行计算了（若能计算出子节点范围超过设定大小，再计算优化是最好）

**Comment by Yanyan-Wang** [comment]
暂不支持。只支持设置 padding
